### PR TITLE
Added css minification and javascript uglification to the Gruntfile

### PR DESCRIPTION
**Pull request by margaretjoanmiller**

### CODE REVIEW

**Comment by CreaturePhil** [comment]
I think we want to keep the dist folder.

File: Gruntfile.js [hunk 1]
```diff
@@ -20,9 +38,11 @@ module.exports = function(grunt) {
     }
 
   });
+  grunt.loadNpmTasks('grunt-contrib-cssmin');
```
Comment (CreaturePhil) [inline Gruntfile.js]: you don't need to add these because of the `load-grunt-tasks` module

**Comment by margaretjoanmiller** [comment]
Sorry, I'm not that used to working with Grunt :P

**Comment by gigavinyl** [comment]
Also, can you run the Gruntfile to create the dist folder and commit it please? My grunt install is having issues :/

**Comment by CreaturePhil** [comment]
what is grunt install?

**Comment by margaretjoanmiller** [comment]
I meant the version of grunt I was running. Never mind, I figured it out though.

**Pull request merged**